Client/reasn-client/apps/web/app/Nav.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from './Nav.module.css';
+
+
+type Link = {
+    label: string
+    href: string
+};
+
+const Nav = () => {
+return(
+<body>
+  <header>
+    <nav
+      className="
+        flex flex-wrap
+        items-center
+        justify-between
+        w-full
+        h-20
+        text-lg text-white
+        bg-black
+        rounded-b-2xl
+      "
+    >
+    <div>
+        <a className="block text-white font-bold text-3xl mx-60">Reasn.</a>
+    </div>
+
+    <div className="flex space-x-4 items-center mx-60"> 
```

Comment:
```suggestion
    <div className="flex items-center gap-8"> 
```
Why space-x-4?

Reply:
I guess I wanted to make spaces between logowanie and rejestracja, but changed for gap-8